src/App/HomePage.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/prefer-stateless-function */
+/* Remove after more is added to satisfy eslint */
+import React, { Component } from 'react';
+
+export default class HomePage extends Component {
```

Comment:
@Unesdala please make this a stateless function since it only returns a render, thanks